NEWS.md

```diff
@@ -11,11 +11,21 @@ Refer to the [Release Notes for
 v0.7](https://github.com/JuliaLang/julia/blob/master/HISTORY.md) for a
 detailed list of changes from Julia v0.6.
 
+Standard Library Changes
+------------------------
+
+* The `Libdl` module's methods `dlopen()` and `dlsym()` have gained a
+  `throw_error` keyword argument, replacing the now-deprecated `dlopen_e()`
```

Comment:
Slightly off topic but it seems like this keyword could have been more concisely called `error` or `throw`; `throw_error` feels unpleasantly long. Is it too late to change that?

Reply:
No, it's not, but let's get this merged to fix the regression. We can change the keyword name anytime before the release of 1.1.

Reply:
Right, I don't mean it should be changed in this PR, just a more general remark.